.github/workflows/build.yml

```diff
@@ -0,0 +1,35 @@
+name: PolkadexTEE-worker CI
+
+on:
+  push:
+    branches: [ 'master','develop' ]
+  pull_request:
+    branches: [ 'master','develop' ]
+
+jobs:
+  build-polkadexTEE:
+    runs-on: ubuntu-latest
+    container: "scssubstratee/substratee_dev:1804-2.12-1.1.3-001"
+    steps:
+      - uses: actions/checkout@v2
+      - name: Cache Rust Dependecies
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.cargo/registry
+            ~/.cargo/git
+            target
+            enclave/target
+          key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}
+      - name: Install Rust and WASM Environment
+        run: ./ci/install_rust.sh
+      - name: Information
+        run: cargo --version && rustup show && env
+      - name: Build Project
+        run: make
+      - name: Test Client
+        run: cd enclave && cargo test 2>&1
+      - name: Test Worker
+        run: cd workder && cargo test 2>&1
+      - name: Test Enclave
+        run: cd enclave && cargo test 2>&1
```

Comment:
will not work. But maybe we can use something similiar as in
 https://github.com/apache/incubator-teaclave-sgx-sdk/issues/232 suggested.

Reply:
Yes, So this is wrong right? https://github.com/scs/substraTEE-worker/blob/master/Jenkinsfile#L43

Reply:
Yes, you're right... we'll need to clean that up